### DISCUSSION TITLE
[ FF7 ] BUG at each launch, impossible to use the mod :(

### DISCUSSION THREAD

**Discussion by ALTITUDE67** [julianxhokaxhiu/FFNx#549] [Q&A]
Good morning,
I can't get your mod to run :(
Every time I have this kind of bug and message suddenly I wait before playing my game again
Please explain to me
Cordially

![Capture d’écran 2023-02-27 153327](https://user-images.githubusercontent.com/53009045/221593124-e673ddc5-87c5-4848-a65d-c3194a3a49c6.png)
![Capture d’écran 2023-02-27 152740](https://user-images.githubusercontent.com/53009045/221593135-53beb69c-e43f-403e-818a-4aed3b55cc68.png)

**Comment by eve-atum** [comment]
Your load order is incorrect (press the autosort button on the right, looks like 1/9), then try again.  This also isn't a bug, and is better handled via live chat on either the Tsunamods or Qhimm Discord.
